### DISCUSSION TITLE
Retrieve the process's “workdir” folder.

### DISCUSSION THREAD

**Discussion by amandine-velt** [nextflow-io/nextflow#5163] [Q&A]
Hello everyone, 

I run my nextflow pipeline with the -resume option and a process I run needs to know the task's workdir, which I have to give it as as an option for its personal “workdir”. By task's workdir I mean, for example : 

/path/to/projectdir/work/e5/54dfd7cbb41f1099329d78c0a12bb9

I tried several variables :

${PWD} : give /path/to/projectdir/work
$workflow.workDir : give  /path/to/projectdir
${task.workDir} : give the "null" value

Can anyone tell me how to retrieve the exact folder where nextflow launches the task and generates the files?

Thank you very much
Amandine

**Comment by pditommaso** [comment]
Use `\${PWD}`
